app/lib/actions/pulp3/orchestration/content_view_version/export.rb

```diff
@@ -0,0 +1,59 @@
+module Actions
+  module Pulp3
+    module Orchestration
+      module ContentViewVersion
+        class Export < Actions::EntryAction
+          input_format do
+            param :smart_proxy_id, Integer
+            param :exporter_data, Hash
+          end
+
+          output_format do
+            param :exported_file_name, String
+            param :exported_file_checksum, String
+          end
+
+          def plan(content_view_version)
+            unless File.directory?(Setting['pulp_export_destination'])
+              fail ::Foreman::Exception, N_("Unable to export. 'pulp_export_destination' setting is not set to a valid directory.")
+            end
+
+            sequence do
+              smart_proxy = SmartProxy.pulp_master!
+              action_output = plan_action(::Actions::Pulp3::ContentViewVersion::CreateExporter,
+                                     content_view_version_id: content_view_version.id,
+                                     smart_proxy_id: smart_proxy.id).output
+
+              plan_action(::Actions::Pulp3::ContentViewVersion::Export,
+                                     content_view_version_id: content_view_version.id,
+                                     smart_proxy_id: smart_proxy.id,
+                                     exporter_data: action_output[:exporter_data]
+                                     )
+
+              plan_self(exporter_data: action_output[:exporter_data], smart_proxy_id: smart_proxy.id)
+
+              plan_action(::Actions::Pulp3::ContentViewVersion::DestroyExporter,
+                            smart_proxy_id: smart_proxy.id,
+                            exporter_data: action_output[:exporter_data])
+            end
+          end
+
+          def run
+            smart_proxy = ::SmartProxy.find(input[:smart_proxy_id])
+            api = ::Katello::Pulp3::Api::Core.new(smart_proxy)
+            export_data = api.export_api.list(input[:exporter_data][:pulp_href]).results.first
+            output[:exported_file_name], output[:exported_file_checksum] = export_data.output_file_info.first
+          end
+
+          def humanized_name
+            _("Export")
```

Comment:
I'd really like to be more descriptive here. 'Export' says nothing about what's going on; seeing the word Export five times in the task list requires more digging if you're looking for a specific one; what was exported?; etc

Contrast to CV publish: `Publish content view 'Pulp3View'; organization 'Default Organization'` this information is way more useful, especially to someone who just wants to look at the tasks and get a general sense of what's going on, let alone the user trying to find their specific Export task

Reply:
Should be resolved now :) 
```
Export content view version 'hah 1.0'; content view 'hah'; organization 'Default Organization'
```

Reply:
I don't see any change here

Reply:
Oh the change was -> https://github.com/Katello/katello/pull/8850/files#diff-476181c4440753c9b39d487820d7679bR17 to make this work? Are you not seeing this work when you run from the PR?